extensions/amp-analytics/0.1/cookie-writer.js

```diff
@@ -128,12 +128,17 @@ export class CookieWriter {
     for (let i = 0; i < ids.length; i++) {
       const cookieName = ids[i];
       const cookieObj = inputConfig[cookieName];
+      const sameSite = this.getSameSiteType_(
+        // individual cookie sameSite overrides config sameSite
+        cookieObj['sameSite'] || inputConfig['sameSite']
```

Comment:
I start to think if it's the correct decision to follow the camel case naming here.  `SameSite` is the attribute name not `sameSite`. Worth consider supporting both. WDYT

Reply:
1) Done. 
2) I followed camelCase for consistency since cookies.js allows specifying `options.sameSite`. However, supporting both also makes sense, so I updated the code to allow `sameSite` and `SameSite` (in that order) and added a test for the same.